src/js/disability-benefits/526EZ/components/DisabilityWizard.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import appendQuery from 'append-query';
+
+import RequiredLoginView from '../../../common/components/RequiredLoginView';
+import ErrorableCheckboxes from '../../../common/components/form-elements/ErrorableCheckboxes';
+import ErrorableRadioButtons from '../../../common/components/form-elements/ErrorableRadioButtons';
+
+import { toggleLoginModal } from '../../../login/actions';
+
+import { disabilityStatusOptions, disabilityUpdateOptions, layouts } from '../helpers';
+
+const { chooseStatus, chooseUpdate, applyGuidance } = layouts;
+
+class DisabilityWizard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentLayout: chooseStatus
+    };
+  }
+
+    getButtonContainer = () => {
+      const { currentLayout, disabilityStatus, add, increase } = this.state;
+      const { profile, loginUrl, verifyUrl } = this.props;
+      const notUpdatingStatus = disabilityStatus === 'first' || disabilityStatus === 'appeal';
+      const updatingStatus = disabilityStatus === 'update';
+      const eligibleForIncrease = updatingStatus && !add && increase;
+      const ineligibleForIncrease = updatingStatus && add;
+      const atIncreaseGuidance = currentLayout === applyGuidance && eligibleForIncrease;
+      const atEbenefitsGuidance = currentLayout === applyGuidance && (notUpdatingStatus || ineligibleForIncrease);
+
+      return  (<div>
+        {/* TODO: Check with @goldenmeanie that users shouldn't be able to go back from third page */}
```

Comment:
@goldenmeanie Back buttons have been provided on all but the first page of the wizard, does this seem helpful?

Reply:
Yep!